### ISSUE TITLE
Probably UTF related problem with translating

### ISSUE THREAD

**Issue by dk-cze** [Irrelon/jquery-lang-js#55]
Hello,

I have this en.json:

{
  "token": {
    "Cestina": "Czech",
    "Čeština": "Czech",
    "Anglictina": "English",
    "Angličtina": "English"
  },
  "regex": [
    [
      "My regex",
      "i",
      "someReplacement"
    ]
  ]
}

This select inside my HTML page:

<select name="testSelect">
<option lang="cs" value="1" onclick="window.lang.change('cs'); return false;">Čeština</option>
<option lang="cs" value="2" onclick="window.lang.change('en'); return false;">Angličtina</option>
/select>

and they dont work with this in my console:

Translation for "Angličtina" not found in language pack: en
jquery-lang.js (řádek 559)
Translation for "Čeština" not found in language pack: en
jquery-lang.js (řádek 559)

and when I change my text to "Anglictina" and "Cestina" then the translations work again...

What should I do? How to change encoding to use special characters of Czech language?

Many thanks.
Have a nice day.

**Comment by Irrelon** [comment]
Hey ya, that should work, are you sure you are saving your JSON file as UTF-8 encoded and not another format?

**Comment by Irrelon** [comment]
Also what text editor are you using to edit the JSON?

**Comment by dk-cze** [comment]
Hello,
I found that it is maybe some kind of bug inside Jetbrains WebStorm editor which I am using... I have set UTF8 as project and IDE file encoding, but it still saves JSON files as ASCI... after every edit when I open my JSON in Notepad++ to check it I have to manually convert to UTF8 to get my application running again (in other case I get errors again).
I am sorry for wasting your time.
Thanks for help and have a nice day. :)
